src/Tools/IdeBenchmarks/IdeBenchmarks.csproj

```diff
@@ -12,7 +12,6 @@
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="$(BenchmarkDotNetVersion)" />
     <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="$(BenchmarkDotNetDiagnosticsWindowsVersion)" />
-    <PackageReference Include="Microsoft.Build.Locator" Version="$(MicrosoftBuildLocatorVersion)" />
```

Comment:
This PR generally goes through the repo and removes uses of MSBuild Locator that was used in places that were using MSBuildWorkspace. Those just aren't necessary anymore.